NOTICE.bin

```diff
@@ -913,3 +913,6 @@ Third-Party Notices and Licenses:
 - Hadoop: Apache Hadoop is used as a dependency
   License: Apache License 2.0
   Source/Reference: https://github.com/apache/hadoop/blob/trunk/NOTICE.txt
+
+OpenLineage
+Copyright 2018-2025 The Apache Software Foundation
```

Comment:
Open Lineage is not an ASF project so this statement is not correct

Reply:
Any change here would also need to be added to the NOTICE file as Open Lineage code is included in the source release

Reply:
update with `OpenLineage  License: Apache 2.0 license`, is it enough?

Reply:
It needs to be what is relevant from Open Lineage's NOTICE file. Copyright lines are generally only added to NOTICE files if they are relocated from a file headers. There's no need to repeat license information that has already been listed in the LICENSE file. Please see https://infra.apache.org/licensing-howto.html

Reply:
updated with OpenLineage NOTICE file content